models/comments-model.js

```diff
@@ -45,3 +45,19 @@ exports.createComments = (articleId, { username, body }) => {
       return comment;
     });
 };
+
+exports.removeComment = (commentId) => {
+  if (!Number.isInteger(Number(commentId))) {
+    return Promise.reject({ status: 400, msg: "Bad request" });
+  }
+  return exports
+    .selectComments(commentId)
```

Comment:
There's no need to use this. You can use the RETURNING* on your query. If there are no rows, then that would be a 404 for you to reject.

Reply:
<img width="839" alt="Screenshot 2024-08-27 at 14 20 15" src="https://github.com/user-attachments/assets/0a8dc70a-e199-45d9-a2ed-e4a0770df834">